spkrepo/tests/common.py

```diff
@@ -362,7 +362,7 @@ def login(self, email, password):
         return self.client.post(
             url_for("security.login"),
             data=dict(email=email, password=password),
-            follow_redirects=True,
```

Comment:
The tests fail because of this: https://github.com/SynoCommunity/spkrepo/commit/76951e0e5850504b16c9164b97a23883488269d6

This needs to be a real generated value. I just don't know how to create them. Using fake ones doesn't work

Reply:
This section for the login action was failing for two reasons:
1. The user being created did not have a hashed password (fixed in https://github.com/SynoCommunity/spkrepo/pull/102/commits/39c9c88a3dbace5e8cf63087a365ba2f720bc7d6)
2. The login process needed to use the non-hashed password for successful login (fixed in https://github.com/SynoCommunity/spkrepo/pull/102/commits/c43aeb8940f6337f01e594ea8866b47c8ef3f2f7)

This was seen in my tests where the login action resulted in a status 200 rather than a redirect. Adding some print statements to show the body of the output this confirmed the invalid password for login. Once corrected, all tests involving `logged_user` succeeded.